client/src/styles/components/Header.style.js

```diff
@@ -2,15 +2,15 @@ import styled from 'styled-components';
 import { Link } from 'react-router-dom';
 
 export const StyledHeader = styled.div`
-    height: 56px;
-    position: fixed;
+    height: 3.5rem;
+    position: sticky;
```

Comment:
The Projects and Volunteer pages need the header to be `fixed`. Could you either work that into your styling on this page, or else create a variable that you can pass to `Header` to swap between the two based on what page it is?